selfdrive/ui/ui.cc

```diff
@@ -157,6 +157,22 @@ static void update_model(UIScene &scene, const cereal::ModelDataV2::Reader &mode
   update_line_data(scene, model.getPosition(), 0.5, 0, &scene.track_vertices, path_length);
 }
 
+static void update_lead(const cereal::RadarState::LeadData::Reader &lead,
```

Comment:
@HaraldSchafer: This function is still in the draft stage, do you think it is correct?

Reply:
Looks correct to me!